command-expansion-server/src/app.ts

```diff
@@ -571,11 +577,26 @@ app.post('/get-seqjson-for-seqid-and-simulation-dataset', async (req, res, next)
     };
   });
 
+  const errors = sortedSimulatedActivitiesWithCommands.flatMap(ai => ai.errors ?? []);
```

Comment:
Why do we only need to flatten the `errors` array here? Is there something special about what it might (or does) look like in this method?

Reply:
Correct.